docs/configuration.rst

```diff
@@ -169,6 +169,7 @@ The ``LETSENCRYPT_SHORTCHAIN`` (default: False) setting controls whether we send
 
 .. _`android handsets older than 7.1.1`: https://community.letsencrypt.org/t/production-chain-changes/150739
 
+The ``REAL_IP_HEADER`` (default: unset) and ``REAL_IP_FROM`` (default: unset) settings controls whether HTTP headers such as ``X-Forwarded-For`` or ``X-Real-IP`` should be trusted. The former should be the name of the HTTP header to extract the client IP address from and the later a comma separated list of IP addresses designing which proxies to trust. If you are using Mailu behind a reverse proxy, you should set both. Setting the former without the later introduces a security vulnerability allowing a potential attacker to spoof his source address.
```

Comment:
Since the real client ip is only used if both settings are set, we must say must instead of should.
> if you are using Mailu behind a reverse proxy, you **must** set both. Setting the former without the later introduces a security vulnerability allowing a potential attacker to spoof his source address. The source client ip address from the reverse proxy is only used if both settings (REAL_IP_HEADER and REAP_IP_FROM) are set in mailu.env.

Reply:
Hmm, no; If you set REAL_IP_HEADER, the nginx module will replace $remote_addr (see https://github.com/Mailu/Mailu/blob/1d9850490c05828aa9e310f64578a189ceecfe8d/core/nginx/conf/nginx.conf#L23)... so the documentation is accurate; You can get away with setting only this one (but shouldn't for security reasons).

X-Forwarded-For will be mangled (sanitized as what's in it can't and shouldn't be trusted) but it will work.

Do you want me to actually enforce that both are set to actively prevent insecure configurations?